cookbook/https.rst

```diff
@@ -0,0 +1,35 @@
+HTTPS
+=====
+
+Sulu behave the same way as Symfony when generating urls.
+If the requested protocol is ``https`` it will use automatically ``https`` to generate its urls.
+
+If this does not happen its a hint that you are behind a load balancer or have a proxy between your request server.
+In that case you should have a look at the `Symfony Proxy Documentation`_ and configure your trusted proxy ip addresses.
```

Comment:
"In that case you should add the IP address of your trusted proxy as explained in the `Symfony Proxy Documentation`_."